### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by MICHAELKITH**
### **In this milestone:**
**I have commited;**
1. index.html file
2. style.css file
3. images files

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+  margin-top: 2rem;
+  display: grid;
+  grid-template: repeat(4, 25%) / auto;
+  padding: 17% 5%;
+  gap: 50px;
+}
+
+.personal-info {
+  margin-top: 4rem;
+}
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [ ] Great job 👍. Kindly make sure your `About Myself` text size looks as close as possible to your chosen Figma design 🙏 
- [ ] Kindly make sure you give your text the correct color `#172B4D`

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+
+.more-button,
+.check-button {
+  border: 1px solid #6070ff;
+  border-radius: 10px;
+  width: 124px;
+  height: 48px;
+  padding: 13px;
+  gap: 10px;
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [ ] Kindly give your `button` more margin as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+li {
+  list-style: none;
+}
+
+.container {
+  width: 100%;
+  margin: 0 auto;
+  max-width: 380px;
+  border: 1px solid #7f8cff;
+  background-color: var(--container-bg);
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [ ] Kindly remove the `border` around your design as it isn't in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+}
+
+.more-button-1 {
+  border: 1px solid #6070ff;
+  border-radius: 10px;
+  width: 128px;
+  height: 48px;
+  padding: 13px;
+  gap: 10px;
+  background-color: var(--button-color);
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [ ] Kindly give all your `buttons` interaction as seen in your chosen Figma design 🙏 

image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @MICHAELKITH , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @MICHAELKITH 
### I'm Levy, your PR reviewer

 Excellent, you nailed it :sunglasses: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 



## Highlights
- Good UI :heavy_check_mark: 
- All linters are passing :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MICHAELKITH** [comment]
@levy002 Good Job!

**Pull request merged**